react_ws_src/src/helpers/computer_moves.js

```diff
@@ -0,0 +1,168 @@
+const BOARD_SIZE = 9;
+const CELL_PREFIX = 'c';
+const PLAYER = 'x';
+const COMPUTER = 'o';
```

Comment:
These would be better off being driven from some kind of `globals` or `config` so that they are consistent throughout the app, but I didn't want to start making the changeset even bigger. It could be done easily in another PR.